### ISSUE TITLE
Hamburger menu has no alt text, just "button"

### ISSUE THREAD

**Issue by alexrcabral** [benetech/MathShare#684]
Problem: The current hamburger menu (when the screen is below a certain width) is not labelled. This can be confusing for screen reader users. 

Solution: The hamburger menu button should be labelled "Menu"

**Comment by rupeshparab** [comment]
Hamburger menu has been removed, I am moving this task to the QA queue.
Let me know if anything else needs to be done for this task.
